print_nanny_webapp/alerts/models.py

```diff
@@ -82,6 +93,42 @@ class AlertSettings(PolymorphicModel):
     )
 
 
+class MethodSettings(PolymorphicModel):
+    MethodChoices = Alert.AlertMethodChoices
+    method = models.CharField(choices=MethodChoices.choices, max_length=255)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, db_index=True)
+    enabled = models.BooleanField(
+        default=True, help_text="Enable or disable this alert delivery method"
+    )
+    created_dt = models.DateTimeField(auto_now_add=True, db_index=True)
+    updated_dt = models.DateTimeField(auto_now=True, db_index=True)
+
+
+##
+# Method Settings models
+##
+
+
+class DiscordMethodSettings(MethodSettings):
+    # Channel ID is where to send a message if you're not going for event-driven
+    # Channel IDs are of "snowflake" type
+    # Snowflakes are 64 bit unsigned integers represented as decimal strings
+    # so max 21 characters - 24 to be "safe"
+    # https://discord.com/developers/docs/reference#snowflakes
+    # TODO: Add a label with capital 'ID'. `label` kwarg is not recognized?
+    channel_ids = ArrayField(models.CharField(
```

Comment:
Do you anticipate needing to lookup a row by an individual channel_id or user_id? It seems to me like this is the only way to map an incoming message (from Discord) back to a `User` data structure. Let me know if I'm missing something obvious!

Reply:
huh, I didn't think this far. ArrayField lookups are expensive I imagine, right?

Reply:
Yeah, the database engine doesn't do great at look-ups against Array-structured fields. No worries though! 

How about we keep this simple, and only permit the user to set one channel id. Since this is a nights/weekends passion project, I'm favoring lean functionality until someone explicitly asks for something more robust (like multiple channel support). That would give us a model like...

```
class DiscordSettings(models.Model):
    created_dt
    updated_dt
    user = models.OnetoOneField
    channel_id = models.BigIntegerField
    channel_name = models.CharField
```